coremain/run.go

```diff
@@ -49,15 +48,8 @@ func Run() {
 		log.SetOutput(os.Stdout)
 	case "stderr":
 		log.SetOutput(os.Stderr)
-	case "":
-		log.SetOutput(ioutil.Discard)
 	default:
-		log.SetOutput(&lumberjack.Logger{
-			Filename:   logfile,
-			MaxSize:    100,
-			MaxAge:     14,
-			MaxBackups: 10,
-		})
+		log.SetOutput(ioutil.Discard)
```

Comment:
Thanks @miekg. The PR has been updated. Please take a look and let me know for any issues.